### DISCUSSION TITLE
信通院 - 开源合规指南（企业篇）发布

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#97] [资料分享]
近年来，开源生态发展势头迅猛，在推动技术创新、促进产业协作、加快各行业数字化进程方面发挥的作用日益凸显。企业或个人在使用、参与或主导开源项目的过程中，一般都会涉及开源许可证的相关问题。开源许可证是一种允许源代码、蓝图或设计等在特定条款下被使用、修改和共享的许可协议。开源许可证授予了被许可人广泛的权利，但是也要求其应承担一定的合规义务。如果对开源许可证的理解或适用上存在误区，可能会产生诸多风险，如：被迫开放源代码的风险、许可证的兼容性风险、违约风险及知识产权风险等。

为了帮助企业规避开源风险，建立完善的开源治理体系。中国信通院联合企业用户、科研机构、法律专家于5月20日召开的“OSCAR开源先锋日”上筹备成立【可信开源合规计划】。【可信开源合规计划】为聚焦开源合规的国内开源组织，旨在凝聚各方力量，聚集一大批国内开源合规人才，整合优质资源，完善开源合规标准体系，输出开源合规建设经验，全面提升我国开源合规水平，为行业的发展提供强劲合力。目前共有53家首批成员单位，正式成员为22家，预备成员为31家。

全文：https://xlab2017.yuque.com/msdpvs/rpifg5/26197083

**Comment by wangleo61** [comment]
给力王老师，我们也有参与


发件人: Will ***@***.***>
发送时间: Tuesday, May 24, 2022 3:23 PM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] 信通院 - 开源合规指南（企业篇）发布 (Discussion #97)


近年来，开源生态发展势头迅猛，在推动技术创新、促进产业协作、加快各行业数字化进程方面发挥的作用日益凸显。企业或个人在使用、参与或主导开源项目的过程中，一般都会涉及开源许可证的相关问题。开源许可证是一种允许源代码、蓝图或设计等在特定条款下被使用、修改和共享的许可协议。开源许可证授予了被许可人广泛的权利，但是也要求其应承担一定的合规义务。如果对开源许可证的理解或适用上存在误区，可能会产生诸多风险，如：被迫开放源代码的风险、许可证的兼容性风险、违约风险及知识产权风险等。

为了帮助企业规避开源风险，建立完善的开源治理体系。中国信通院联合企业用户、科研机构、法律专家于5月20日召开的“OSCAR开源先锋日”上筹备成立【可信开源合规计划】。【可信开源合规计划】为聚焦开源合规的国内开源组织，旨在凝聚各方力量，聚集一大批国内开源合规人才，整合优质资源，完善开源合规标准体系，输出开源合规建设经验，全面提升我国开源合规水平，为行业的发展提供强劲合力。目前共有53家首批成员单位，正式成员为22家，预备成员为31家。

全文：https://xlab2017.yuque.com/msdpvs/rpifg5/26197083

―

**Reply by will-ww** [reply]
嗯，抽空分享下哈~

**Comment by wangleo61** [comment]
好呀


发件人: Will ***@***.***>
发送时间: Wednesday, May 25, 2022 10:39 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] 信通院 - 开源合规指南（企业篇）发布 (Discussion #97)


嗯，抽空分享下哈~

―

**Comment by ChingOK** [comment]
![image](https://github.com/user-attachments/assets/c66e3cb7-788c-48a2-a716-b0f80aa16326)
显示没有权限，联系管理员。
